client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import "react-calendar/dist/Calendar.css";
+import "./CalendarOverride.css";
+import Calendar from "react-calendar";
+import styles from "./CalendarPage.module.css";
+import TaskCard from "../../components/TaskCard/TaskCard.js";
+import CalendarHeader from "../../components/CalendarHeader/CalendarHeader.js";
+
+const dayLabelFormatter = (locale, label) => {
```

Comment:
Locale doesn't seem to be used here

Reply:
I would also rename the function to `dayAbbreviation` or something that's a bit more discernable than dayLabelFormatter

Reply:
`locale` is part of the function signature `react-calendar` accepts. See "formatShortWeekday" in [react-calendar](https://www.npmjs.com/package/react-calendar).

I've changed `locale` to `_` to indicate this variable is unused. I've also renamed the function to `getDayAbbreviation`.

Reply:
Ah, I see. That makes sense!